.github/workflows/wheels-manylinux-build.yml

```diff
@@ -0,0 +1,201 @@
+        default: ''
+      skbuild-build-options:
+        required: false
+        type: string
+        default: ''
+
+jobs:
+  wheel-epoch-timestamp:
+    name: wheel epoch timestamper
+    runs-on: ubuntu-latest
+    outputs:
+      rapids_epoch_timestamp: ${{ steps.get_epoch_time.outputs.rapids_epoch_timestamp }}
+
+    steps:
+    - name: Get epoch timestamp to use as nightly version
+      id: get_epoch_time
+      run: echo "RAPIDS_EPOCH_TIMESTAMP=$(date +%s)" >> "${GITHUB_OUTPUT}"
+
+  wheel-build:
+    name: cibuildwheel ${{ matrix.arch }} ${{ matrix.python }} ${{ matrix.ctk }}
+    needs: wheel-epoch-timestamp
+    strategy:
+      matrix:
+        python: ['3.8', '3.10']
+        arch: [amd64, arm64]
+        ctk: ['11.8.0', '12.0.0']
+    runs-on:
+      - self-hosted
+      - linux
+      - ${{ matrix.arch }}
+      - cpu16
+    container:
+      # ctk version of the cibw container is irrelevant in the manylinux case
+      # it's simply a launcher for cibuildwheel
+      image: "rapidsai/cibuildwheel:cuda-runtime-11.8.0-ubuntu20.04"
+      volumes:
+        - /var/run/docker.sock:/var/run/docker.sock
+      env:
+        RAPIDS_BUILD_TYPE: ${{ inputs.build_type }}
+        RAPIDS_PY_VERSION: ${{ matrix.python }}
+        AWS_ACCESS_KEY_ID: ${{ secrets.RAPIDSAI_GHA_AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.RAPIDSAI_GHA_AWS_SECRET_ACCESS_KEY }}
+
+    steps:
+    - name: checkout code repo
+      uses: actions/checkout@v3
+      with:
+        repository: ${{ inputs.repo }}
+        ref: ${{ inputs.sha }}
+        fetch-depth: 0 # unshallow fetch for setuptools-scm
+        persist-credentials: false
+
+    - name: Standardize repository information
+      run: |
+        echo "RAPIDS_REPOSITORY=${{ inputs.repo || github.repository }}" | tee --append "${GITHUB_ENV}" "${GITHUB_OUTPUT}"
+        sha="${{ inputs.sha }}"
+        if [[ "${sha}" == "" ]]; then
+          sha=$(git rev-parse HEAD)
+        fi
+        echo "RAPIDS_SHA=${sha}" | tee --append "${GITHUB_ENV}" "${GITHUB_OUTPUT}"
+        echo "RAPIDS_REF_NAME=${{ inputs.branch || github.ref_name }}" | tee --append "${GITHUB_ENV}" "${GITHUB_OUTPUT}"
+        echo "RAPIDS_NIGHTLY_DATE=${{ inputs.date }}" | tee --append "${GITHUB_ENV}" "${GITHUB_OUTPUT}"
+
+    - name: Preprocess extra repos
+      id: preprocess-extras
+      if: ${{ inputs.extra-repo != '' }}
+      run: |
+        EXTRA_REPO_PATH=$(echo ${{ inputs.extra-repo }} | cut -d "/"  -f 2)
+        echo "EXTRA_REPO_PATH=${EXTRA_REPO_PATH}" >> $GITHUB_OUTPUT
+
+    - name: checkout extra repos
+      uses: actions/checkout@v3
+      if: ${{ inputs.extra-repo != '' }}
+      with:
+        repository: ${{ inputs.extra-repo }}
+        ref: ${{ inputs.extra-repo-sha }}
+        path: ${{ steps.preprocess-extras.outputs.EXTRA_REPO_PATH }}
+        ssh-key: ${{ secrets[inputs.extra-repo-deploy-key] }}
+        persist-credentials: false
+
+    - name: Set CTK-related vars from input CTK versions
```

Comment:
Another step that could maybe be shared across workflows. I see that the wheel test only uses the wheel name and not the cuda suffix, so could make sense to split that up, although it might also be fine to just have the extra var floating around in that case.

Reply:
Actually, in setting the stage to support CTK 12 in the future, I've gone around appending `_cu11` (or `_cu${ctk_major}`) to the `package-name` i.e. `RAPIDS_PY_WHEEL_NAME`

Old (main branch) workflow:
* RMM: rmm_wheel_python_rmm_x86_64_3.8.tar.gz

New (this WIP PR) workflow, before CTK 12 is added:
* RMM: rmm_wheel_python_rmm_cu11_x86_64_3.8.tar.gz

Future (when CTK 12 is added):
* RMM: rmm_wheel_python_rmm_{cu11, cu12}_x86_64_3.8.tar.gz